src/app/_components/DistrictSearch/DistrictSearch.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import { cn } from '@utils/cn';
+import { Heading } from '@/app/_components';
```

Comment:
🟠 import it directly? @components/Typography

Reply:
You sure it will work?

![image](https://github.com/keenethics/mayak/assets/103147058/cfc7a472-62ac-4cbc-b49d-a641bb8b5888)

Reply:
✅

Reply:
Though it's a little confusing